### ISSUE TITLE
Help needed to configure Waf2Py properly

### ISSUE THREAD

**Issue by AppLEaDaY** [ITSec-Chile/Waf2Py#17]
I installed the software correctly, though now I see quite a lot depends on my deduction.
The scenario where I might want to use Waf2Py is made of a nginx server working as a proxy for Docker containers.
So the _application_ should be the name for the nginx virtual host, _Listens to_ should be the public IP address bound to that name and _Real web server_ should be the IP address of the container where the nginx virtual host would forward the requests to: **is this correct?**
I had a look at the _Expert Configuration_ tab, but in the _Nginx configuration_ section I find hard to interpret the _proxy_pass_ line.
Anyway I clicked on the _Enable application_ button, but the status of the _website_ stays _Disabled_.
What am I missing?

Thanks in advance!

Andrea

**Comment by ch-rigu** [comment]
yes, "Listen to" is the public address and the container address in "Real Application IPs". That need to configured under "HTTP/HTTPS" tab. Then press the "play" button and the app will be enabled.

Chris.

**Comment by AppLEaDaY** [comment]
I press the play buttons, but every "app" I configured there stays disabled.
Actually I have no real container listening behind nginx, but I expect nginx virtual hosts could be enabled anyway.
How can I debug what's happening, or what is not happening?
Thanks in advance!

Andrea

![image](https://github.com/ITSec-Chile/Waf2Py/assets/10222899/99fe552f-ede0-49c4-8016-9b4aa907615f)
